static/layout.less

```diff
@@ -43,6 +43,14 @@
 }
 #toc-sidebar {
   grid-area: toc-sidebar;
+  overflow: auto;
```

Comment:
This shouldn’t be necessary. The overflow is already set here: https://github.com/canjs/bit-docs-html-canjs/blob/f5fe51a08ec578aabb60a2119ff3e6f452e38689/static/layout.less#L67